cryptographic_estimators/MQEstimator/mq_problem.py

```diff
@@ -36,12 +36,26 @@ class MQProblem(BaseProblem):
     - ``n`` -- number of variables
     - ``m`` -- number of polynomials
     - ``q`` -- order of the finite field (default: None)
+    - ``theta`` -- exponent of the conversion factor (default: 2)
     - ``nsolutions`` --  number of solutions in logarithmic scale (default: max(expected_number_solutions, 0))
-    - ``memory_bound`` -- maximum allowed memory to use for solving the problem
+    - ``memory_bound`` -- maximum allowed memory to use for solving the problem (default: inf)
+
+    NOTE:
+
+    - If ``0 <= theta <= 2``, every multiplication in GF(q) is counted as `log2(q) ^ theta` binary operation.
+    - If ``theta = None``, every multiplication in GF(q) is counted as `2 * log2(q) ^ 2 + log2(q)` binary operation.
+
```

Comment:
shouldn't this comment added to the field `theta`, just above?

Reply:
I think it is too long to be added there. What do you think?

Reply:
I mean:  it should be possible to write something like:
```
    - ``theta`` -- exponent of the conversion factor (default: 2)
         - If ``0 <= theta <= 2``, every multiplication in GF(q) is counted as `log2(q) ^ theta` binary operation.
         - If ``theta = None``, every multiplication in GF(q) is counted as `2 * log2(q) ^ 2 + log2(q)` binary operation.
    - ``nsolutions`` --  number of solutions in logarithmic scale (default: max(expected_number_solutions, 0))
```

Reply:
Agree